src/style/app.less

```diff
@@ -549,6 +549,10 @@ input[type=text][readonly] {
   }
 }
 
+#search-container.embed-search {
+  display: none;
+}
```

Comment:
Remove this css and move it in the [.embed section](https://github.com/geoadmin/mf-geoadmin3/blob/b89c4a4c3750932c0fc3eb2015504774a3f75a19/src/style/app.less#L1748)